packages/forms/src/helpers.php

```diff
@@ -10,7 +10,7 @@ function array_move_after(array $array, $keyToMoveAfter): array
         $indexToMoveAfter = array_search($keyToMoveAfter, $keys);
         $keyToMoveBefore = $keys[$indexToMoveAfter + 1] ?? null;
 
-        if (filled($keyToMoveBefore)) {
```

Comment:
Please explain this change

Reply:
Ah, forget it. I thouht it would be more readable.